plugins/scm.js

```diff
@@ -12,6 +12,12 @@ const pipelineId = Joi.reach(models.pipeline.base, 'id').optional();
 const prNum = Joi.reach(core.scm.hook, 'prNum').allow(null).optional();
 const scmContext = Joi.reach(models.pipeline.base, 'scmContext').optional();
 const scmUri = Joi.reach(models.pipeline.base, 'scmUri').required();
+const scmInfo = Joi.object({
```

Comment:
Why not just do `.optional()` here? Also, can we have a test for this change?

Reply:
@minz1027 Thanks for the comment! I have fixed it